integration-tests/tests/network/runner.rs

```diff
@@ -230,7 +230,7 @@ impl StateMachine {
                       _ctx: &mut Context<WaitOrTimeoutActor>,
                       _runner| {
                     if can_write_log.swap(false, Ordering::Relaxed) == true {
-                        debug!(target: "network", message = "runner.rs: Action", num_prev_actions, action = ?action_clone);
+                        debug!(target: "network", num_prev_actions, action = ?action_clone, "runner.rs: Action");
```

Comment:
Note: we don't need to put the name of the file (`runner.rs`) into the logging call. Each logging event carries metadata, which contains full path to file:

https://docs.rs/tracing-core/0.1.21/tracing_core/event/struct.Event.html#method.metadata

If we need location info, we can adjust tracing subscriber to print file name and column.

Reply:
Eg, see the Pretty format here: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/index.html

```
Oct 24 12:57:29.386 fmt_pretty::yak_shave: shaving yaks
  at examples/examples/fmt/yak_shave.rs:38 on main
  in fmt_pretty::yak_shave::shaving_yaks with yaks: 3
```

Reply:
@matklad If we do that, we would be adding the file name to all logs, right? Usually, that's not needed, I just need it for this file. I'm grepping for `runner.rs` to find list of all actions executed during this test.

Reply:
Just having `Oct 24 12:57:29.386 fmt_pretty::yak_shave: shaving yaks` would be enough.

At least having `` fmt_pretty::yak_shave```  is useful. Otherwise, I had to rely on the message name to find the right log.